cryptoinvestor/models/user.py

```diff
@@ -7,3 +8,44 @@ def __init__(self, *, username: str, password: str, account: Account):
         self.username = username
         self.password = password
         self.account = account
+        self.id = id
+        self.rates = {}
+        self.coin_status = 10000
+        self.bought = dict()
+
+    def buy(self) -> bool:
+        count = request.args.get('count')
+        count = float(count)
```

Comment:
You can do this in one line. Also please add a default value to `get` method something like:

```python
count = float(request.args.get('count', 0))
```

Reply:
done